### DISCUSSION TITLE
[BUG] 2nd Playback freezes entire OS.

### DISCUSSION THREAD

**Discussion by RiaTheFae** [jurialmunkey/skin.arctic.horizon.2#1036] [Discussion]
Heya! I'm looking for some support, I've had this issue for a long time, I've tried searching for solutions but I could never find one. 
I can start playback completely fine, the issue is when the playback stops and I try to start a new one, the entire OS freezes(LibreELEC v12.0.0).

Everything works fine if I use a different skin but AH2 causes these issues.

[Debug Log](https://paste.kodi.tv/afekexefij.kodi).

**Comment by josephruggiero** [comment]
I'm running on a rPi4 and had the same issue.

Switching my render method from "Direct-to-Plane" to "EGL" made things a lot more stable for me.
[Settings->Player->Videos](https://kodi.wiki/view/Settings/Player/Videos#Processing)


Some of my 4k HDR content has washed out colors and stutters with EGL selected, but overall it's been a much more stable experience.

**Comment by TimboZero** [comment]
@AerYareli your log is too much to be useful.
Reboot Kodi twice, then play pack something and another something to cause issue. After this even whilst Kodi is frozen FTP in and grab the log and post that one.
Also get rid of Fen it’s a dead addon and the developer has stopped working on it and replaced it with Fenlight which is much better much faster, much more stable and is still updated regularly.
